cmd/kubeadm/app/master/manifests.go

```diff
@@ -323,9 +329,11 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 		"requestheader-extra-headers-prefix": "X-Remote-Extra-",
 		"requestheader-client-ca-file":       path.Join(cfg.CertificatesDir, kubeadmconstants.FrontProxyCACertName),
 		"requestheader-allowed-names":        "front-proxy-client",
+	}
+	if k8sVersion.AtLeast(version.MustParseSemantic("v1.7.0")) {
```

Comment:
Will this work for v1.7.0-alpha.0 as well?
Can't you just say `k8sVersion.AtLeast("v1.7.0-alpha.1")` (and move that constant elsewhere)?